proxies/live/apiproxy/policies/ExtractVariables.GetProofingLevel.xml

```diff
@@ -0,0 +1,13 @@
+<ExtractVariables name="ExtractVariables.GetProofingLevel" continueOnError="true">
+  <!-- The source element points to the variable populated by AccessEntity policy.
+  The format is <policy-type>.<policy-name>.
+  In this case, the variable contains the whole developer profile. -->
+  <Source>AccessEntity.AccessEntity.GetApp</Source>
+  <VariablePrefix>developer.app</VariablePrefix>
+  <XMLPayload>
+    <Variable name="nhs-login-allowed-proofing-level" type="string">
+      <!-- You parse elements from the developer profile using XPath. -->
+      <XPath>/App/Attributes/Attribute[Name='nhs-login-allowed-proofing-level']/Value/text()</XPath>
```

Comment:
Should this parameter name nhs-login-minimum-proofing-level?

Reply:
no, spec says the 'custom attribute is :
"nhs-login-allowed-proofing-level"
the variable passed to the backend is:
"developer.app.nhs-login-minimum-proofing-level"

https://nhsd-confluence.digital.nhs.uk/display/DSA/1.+Patient+Vaccine+History#id-1.PatientVaccineHistory-APIMFunctionality

Reply:
thx @vurt007